Monika After Story/game/zz_transforms.rpy

```diff
@@ -170,6 +170,7 @@ transform mas_chlongjump(x, y, ymax, travel_time=1.0):
 
 #START: Transforms for Monika's sprite animations (blinking/winking/tear-specific-blinking)
 transform blink_transform(open_eyes_img, closed_eyes_img):
+    animation
```

Comment:
what does this statement do?

Reply:
Allows transforms to use `at` - animation timebase, which should keep them running on exp changes rather than restarting. And yes this isn't documented anywhere.